nav2_msgs/action/FollowWaypoints.action

```diff
@@ -0,0 +1,8 @@
+#goal definition
+geometry_msgs/PoseStamped[] poses
```

Comment:
We're assuming the waypoints will be visited in the order provided on the array, and that the last waypoint is the final pose. We just need to mention that somewhere.

Reply:
Will add to the eventual docs

Reply:
OK